src/stm32/usbfs.c

```diff
@@ -41,6 +41,11 @@
   #define USB_EPTX_STAT USB_EP_TX_STTX
   #define USB_ISTR_EP_ID USB_ISTR_IDN
   #define USB_CNTR_FRES USB_CNTR_USBRST
+#elif CONFIG_MACH_STM32G4
+  // Transfer memory is accessed with 16bits and contains 16bits of data
+  typedef volatile uint16_t epmword_t;
+  #define WSIZE 2
+  #define USBx_IRQn USB_LP_IRQn
```

Comment:
Please move this next to the definition for STM32F0 and STM32L4.

Reply:
Turns out G4 supports 32 or 16 bit access to a 16 bit register, so this fits better with the STM32F103 defines due to the IRQ naming.